pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -524,6 +524,75 @@ spec:
           workspace: release-workspace
       runAfter:
         - populate-release-notes-images
+    - name: collect-atlas-params
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/collect-atlas-params/collect-atlas-params.yaml
+      params:
+        - name: dataPath
+          value: "$(tasks.collect-data.results.data)"
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-data
+    - name: create-product-sbom
+      when:
+        - input: "$(tasks.collect-atlas-params.results.secretName)"
+          operator: notin
+          values: [""]
+      params:
+        - name: dataJsonPath
+          value: "$(tasks.collect-data.results.data)"
+      taskRef:
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/create-product-sbom/create-product-sbom.yaml
+        resolver: git
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-atlas-params
+        - populate-release-notes-images
```

Comment:
Why not just collect-atlas-params? How does this depend on the release notes images?

Reply:
The package URLs used in the SBOM are generated in that task. Sadly you can't see the dependency, as populate-release-notes-images modifies the data.json in place.

Reply:
gotcha. So this is not in the rh-push-to-registry-redhat-io pipeline anymore because it is only used for content shipped with an advisory?

Reply:
Yes, that's so. I had a bit of a brain fart when I added it to the other pipeline :brain:.